app/lib/forms/syncs/skip.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Forms
+  module Syncs
+    class Skip
+      include Formify::Form
+      include Eventable
+
+      attr_accessor :sync
+
+      validates_presence_of :sync
+
+      validate :validate_not_succeeded
+
+      def save
+        with_advisory_lock_transaction(:syncs, sync) do
+          validate_or_fail
+            .and_then { skip }
+            .and_then { schedule_next_sync }
+            .and_then { success(sync) }
```

Comment:
Is this success different than the ones on lines 31 and 37?

Reply:
Not really.  I just add a final `and_then` to ensure we always return what we expect from the form.  This protects against having to ensure we return the right thing from each method for the final result.